workspaces/Data_unconstrainedFpt_withJets/step0

```diff
@@ -80,6 +80,24 @@ def shift_nonpositive_bins(h, value=0.001, error=0.00001):
     # if hwrsg2.GetBinContent(b+1)<0: # why a different treatment for hwrsg2 ?
     #     print "NEGATIVE VALUES in wrong signal sample!"
 
+def transform_to_0_n_bins(h_orig):
```

Comment:
What is the idea behind this transformation? I understand mapping
the bins from `[m_min, m_max]` to `[0, n]`, but why `[0, n+1]`?